src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,12 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard key={movie.imdbId} movie={movie} />
+      ))}
+    </div>
+  </>
```

Comment:
```suggestion
  <div className="movies">
    {movies.map(movie => (
      <MovieCard key={movie.imdbId} movie={movie} />
    ))}
  </div>
```
There is no need to use React fragment here, as you already have a container